src/scss/message.scss

```diff
@@ -4,41 +4,31 @@
 ////
 
 /// Messages styles
-@mixin oFormsMessage($class: 'o-forms') {
-	.#{$class}__message {
-		@include oColorsFor(o-forms__wrapper-highlight, background);
-		@include oTypographySize($_o-forms-typography-scale-large);
-		max-width: $_o-forms-field-max-width;
-		padding: 0 $_o-forms-padding;
-		margin-bottom: $_o-forms-spacing;
-		margin-left: $_o-forms-padding * -1;
-		margin-right: $_o-forms-padding * -1;
-		color: oColorGetColorFor('o-forms__message-default', text);
+@mixin oFormsMessage() {
+	@include oColorsFor(o-forms__wrapper-highlight, background);
+	@include oTypographySize($_o-forms-typography-scale-large);
+	max-width: $_o-forms-field-max-width;
+	padding: 0 $_o-forms-padding;
```

Comment:
these are not `oTypographyMargin/Padding` worthy?

Reply:
This is horizontal spacing which uses the grid gutter :)